src/views/NoteListView.cpp

```diff
@@ -584,7 +477,7 @@ void NoteListView::onSelectFirstNoteEvent()
 
     const QAbstractItemModel * pModel = model();
     if (Q_UNLIKELY(!pModel)) {
-        QNDEBUG(QStringLiteral("No model"));
+        QNERROR(QStringLiteral("No model"));
```

Comment:
Messages printed to error log level better be more verbose. Sure, it's possible to look at the source file and line number to figure out where such log entry comes from but it'd be more convenient if such error message was pretty clear on its own, like "No model is set to NoteListView on select first note event".

Reply:
Yes agree. Just in this particular case, I think its very unlikely the model to be null. And if this happen, then something must be really terribly bad (like design error or like this).

But on general case you are right.
No problem with improving the message.